designs/developer/health checks/health checks-45.md

```diff
@@ -0,0 +1,55 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Health Checks
+Health Checks allow users to detect and handle unhealthy containers in OpenShift.  There are 3 probes that currently exist: Readiness, Liveness, and Startup. A probe is a Kuberbetes action that periodically performs diagnostics on a running container.
+
+## Health Check Configuration
+If there is only one container that doesn’t have any health checks configured, there will be an inline alert with the message shown below and a link to add health checks. If the user clicks the ‘x’ on the inline notification, this message won’t be shown again for this specific deployment/deployment config, but will show on other deployments until the user dismisses it.
+![hc-sidepanel-1conatiner](../health checks/img/hc-sidepanel-1conatiner.png)
+
+This inline alert would also be displayed on the deployment details page.  It would be under the Container header before the list of the containers.
+![dpage1](../health checks/img/dpage 1.png)
+
+Since only one container exists, the user will be brought to this page with the container name listed at the top after “Container”.
+![add-hc-1c](../health checks/img/add-hc-1c.png)
+
+If there is more than one container and any of the containers don’t have health checks configured, there will be an inline alert with the message shown below and a link to add health checks. If the user clicks the ‘x’ on the inline notification, this message won’t be shown again for this specific deployment/deployment config, but will show on other deployments until the user dismisses it.
+![hc-sidepanel-2conatiners](../health checks/img/hc-sidepanel-2conatiners.png)
+![dpage2](../health checks/img/dpage2.png)
+
+Since more than one container exists, the user will be brought to this page with a dropdown menu of all of the containers listed at the top after “Container”.  As the user changes the dropdown, the entire page below will also update.  Once the user begins to edit the form, the dropdown will become disabled.
+![add-hc-2c](../health checks/img/add-hc-2c.png)
+
+
+## Edit Health Checks
+Users can add or edit health checks using the actions dropdown menu.  If there are no health checks configured the action will be “Add Health Checks”. 
+![hc-sidepanel-actionsadd](../health checks/img/hc-sidepanel-actionsadd.png)
+
+If at least one health check is configured, the action will be “Edit Health Checks”.
+![hc-sidepanel-actionsedit](../health checks/img/hc-sidepanel-actionsedit.png)
+
+If there are no health checks configured the title of the form will be “Add Health Checks” and the actions at the bottom will be “Add” and “Cancel”. If only one container exists, that container name will be listed at the top after “Container”.
```

Comment:
We jumped into the edit section but this seems to be repeating information that was described above about Adding Health Checks...am I missing a different context here?